bin/setup

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+##
+# Setup script to symlink configuration and dot files from this repository
+# to proper places in the system.
+#
+# Script is idempotent. It is same to run it several times.
+#
+# TODO:
```

Comment:
TODO what?

Reply:
I'll fill it later a bit. It is to track scripts things which accumulated during development.